cmd/oceantv/broadcast_states.go

```diff
@@ -481,6 +481,19 @@ func (s *directIdle) enter() {
 	s.bus.publish(hardwareStopRequestEvent{})
 }
 
+type directFailure struct {
+	stateFields
+	*broadcastContext `json: "-"`
+}
+
+func newDirectFailure(ctx *broadcastContext) *directFailure {
+	return &directFailure{broadcastContext: ctx}
+}
+func (s *directFailure) enter() {
+	try(s.man.StopBroadcast(context.Background(), s.cfg, s.store, s.svc), "could not stop broadcast on direct failure entry", s.log)
```

Comment:
Does broadcastContext not implement context.Context?

Reply:
nup